package.json

```diff
@@ -4,15 +4,18 @@
   "main": "index.js",
   "version": "0.0.0-development",
   "scripts": {
+    "build": "babel ./index.js -d build -s",
     "commit": "git-cz",
     "check-coverage": "istanbul check-coverage --statements 100 --branches 90 --functions 100 --lines 100",
+    "install": "babel ./index.js -d build -s",
```

Comment:
We shouldn't have an `install` script. Instead, we should add `npm run build` to the `.travis.yml` after [this line](https://github.com/kentcdodds/validate-commit-msg/blob/d8a2d19a387e070ab115683443c8b3397ee1aba0/.travis.yml#L19), then `semantic-release` will publish that.